bin/portageq

```diff
@@ -1425,6 +1433,14 @@ def main(argv):
 
 		os.environ["ROOT"] = root
 
+		if getattr(function, "uses_configroot", False):
+			os.environ["PORTAGE_CONFIGROOT"] = eroot
+			# Disable RepoConfigLoader location validation, allowing raw
```

Comment:
Maybe it would be better:
```
if eroot != "/":
	# Disable RepoConfigLoader location validation...
	portage._sync_mode = True
```

Reply:
I think it is potentially more valuable to return the "raw" data in any case. I think the main reason that RepoConfigLoader checks for existence is to help it portage cope with broken repos.conf, but portageq currently has no need for coping mechanisms here, and we can introduce that kind of thing later if the need arises.